packages/editor/src/extensions/code-block/highlighter.ts

```diff
@@ -198,6 +203,15 @@ export function HighlighterPlugin({
               updated.add(block.pos);
 
               const { id, language } = block.node.attrs;
+
+              const isRegistered =
+                language ? refractor.registered(language) ? true : false : "No need to register"
```

Comment:
You are still pointlessly doing this instead of just `refractor.registered`.

Reply:
OK I am removing it right now.

Reply:
Actually no I checked this check is necessary because when new code block is created language is null so that is why I am doing this check.

Reply:
If language is null, `refractor.registered` will return false, no?

Reply:
It gives error.
![type error](https://user-images.githubusercontent.com/121712508/229887560-a2cfaf16-3375-4289-9e7b-79bdc3c43e2a.PNG)

Reply:
Then just do `language && refractor.registered(language)`

Reply:
Ok